src/App.tsx

```diff
@@ -1,14 +1,26 @@
 import Header from './components/header/Header';
 import ProductTiles from './components/product-tiles/ProductTiles';
 import Footer from './components/footer/Footer';
+import styled from 'styled-components';
+import AppRoutes from './components/routes/AppRoutes';
+import { BrowserRouter } from 'react-router-dom';
+
+const AppContainer = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+`;
 
 function App() {
   return (
-    <>
-      <Header>
-      <ProductTiles />
-      <Footer />
-    </>
+    <BrowserRouter>
```

Comment:
Move this into Routes

Reply:
We're a bit stuck on this, when we move BrowserRouter down into AppRoutes, our AppRoutes tests fail because we're using MemoryRouter, so we get the error message `You cannot render a <Router> inside another <Router>.`